Code/GraphMol/Aromaticity.cpp

```diff
@@ -809,12 +809,18 @@ int aromaticityHelper(RWMol &mol, const VECT_INT_VECT &srings,
       continue;
     }
 
+    auto isDummyAtom = [](const Atom &atom) {
```

Comment:
This is a generally useful function, please put it in QueryOps.h

Reply:
Added.